src/storage/index.ts

```diff
@@ -31,6 +33,19 @@ export interface StorageObject {
   [key: string]: StorageValue;
 }
 
+export function isStorageValue(v: unknown): v is StorageValue {
+  if (isUndefined(v) || isString(v)) {
+    return true;
+  } else if (isObject(v)) {
+    for (const key in v) {
+      return isStorageValue(v[key]);
+    }
+  } else {
+    return false;
+  }
```

Comment:
```suggestion
  if (isUndefined(v) || isString(v)) {
    return true;
  } 
  
  if (isObject(v)) {
    for (const key in v) {
      return isStorageValue(v[key]);
    }
  }

  return false;
```

This seems slightly more readable. What do you think?